modules/ROOT/pages/microsoft-azure-sb-connector.adoc

```diff
@@ -0,0 +1,334 @@
+
+*Reviewers: I see a Lock field but not a Lock Token field. So should I replace Lock Token with Lock. Also, when you say "Do something with the message", can you be more specific?*
+
+The connector processes every message received and acknowledges each message that is processed successfully.
+
+To have the connector automatically acknowledge messages, set the *Receive Mode* field to `AUTOMATIC`.
+
+[[scheduled]]
+=== Scheduled Reception of Messages
+
+The following example configures a Mule app that receives messages from a queue or topic on a specified schedule:
+
+Create a Mule app to receive messages from a queue or topic, as described in <<basic>>.
+
+When you create the app:
+
+* Set the *Unix CRON Expression* field to a CRON expression that follows the UNIX standard. For example, to connect and receive messages every day at 8 AM, use `0 8 * * *`.
+* Set the *Max. Messages to Receive* field to the maximum number of messages for the connector to receive every time the CRON expression triggers a receive.
+
+The connector executes a batch receive every time the CRON expression commands it. Any number of available messages up to the maximum set will be retrieved and processed by the flow.
+
+[[batch]]
+=== Queue Send Batch
+
+The following example configures a Mule app to send multiple messages to a queue:
+
+. Create a Mule app to receive messages from a queue or topic, as described in <<basic>>.
+. Replace the call to the *Publish message* operation with a call to the *Publish batch of messages* Operation. Make sure the *Destination type* and *Destination name* are properly set.
+. When crafting the message, be sure to create a list of strings, since the app will send many messages simultaneously.
++
+*Reviewers: what is the format of the list of strings? Is it comma-separated?*
++
+To split a string by the commas, and send every piece as an individual message, drag a *Transform message* and use the following script:
+```
+%dw 1.0
+%output application/java
+---
+payload splitBy ','
+```
+
+[[enqueuing]]
+=== Schedule the Logical Enqueuing of a Message
+
+The following example configures a Mule app to send a message to all listeners at a specified time:
+
+. Create a Mule app to receive messages from a queue or topic, as described in <<basic>>.
+. Fill in the "Scheduled enqueue time UTC" of the *Publish message* call parameter with the date/time you want the message to be dispatched. For example: `2019-06-27T21:16:46.866Z`.
+
+*Reviewers, where is the Publish message call parameter? I am not seeing it.*
+
+The message will be sent to the destination, which will dispatch the message at the specified time.
+
+== REST operations
+
+* Queue
+** Create
+** Get
+** Get Size
+** List
+** Update
+** Delete
+* Topic
+** Create
+** Get
+** List
+** Update
+** Delete
+* Subscription
+** Create
+** Get
+** Get Size
+** List
+** Update
+** Delete
+* Rule
+** Create
+** Get
+** List
+** Update
+** Delete
+
+== Additional Configuration Information
```

Comment:
As a heading, this really does not tell the audience very much, especially when given at the beginning as part of the topic set content. Can this be made any more specific? 
Also suggest adding an intro that addresses the audience as "you," summarizes/names subtopics, and tells WHY "you" is interested: value statement. GLOBAL in all texts.

Reply:
This heading is in our current connector template. I will save this comment and discuss it with our team.

Reply:
Actually, I thought you were pointing to something else. I added  introductory text and changed the title to "Supported REST Operations."